main.star

```diff
@@ -280,6 +293,7 @@ def run(plan, args={}):
         sovereign_contract_setup_addresses,
         genesis_artifact,
         deploy_optimism_rollup,
+        deployment_stages.get("deploy_cdk_bridge_infra", False),
```

Comment:
here instead of passing `deploy_optimism_rollup` and `deploy_cdk_bridge_infra` as parameters, can you simply pass `deployment_stages`?